src/components/common/SwapPrice.stories.tsx

```diff
@@ -31,3 +32,8 @@ const Template: Story<Partial<Props>> = (props) => {
 
 export const Basic = Template.bind({})
 Basic.args = {} as Props
+
+export const HideBaseToken = Template.bind({})
+HideBaseToken.args = {
+  showBaseToken: false,
+} as Props
```

Comment:
```suggestion
}
```

It will correctly infer types without.